projects/ima-namespace/ima-utils/Makefile

```diff
@@ -7,6 +7,7 @@ DEPS=Dockerfile
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 
 tag: $(DEPS)
+	docker pull linuxkit/$(IMAGE):$(HASH) || \
```

Comment:
Could you use the ORG override pattern we use in all the official packages now? Sorry, things have changed a bit frantically this week